### PR TITLE
Update test for kubeflow 0.5

### PR DESCRIPTION
**Pull request by lluunn**
for https://github.com/kubeflow/kubeflow/issues/2716

cc @richardsliu

### CODE REVIEW

File: testgrid/config.yaml [hunk 1]
```diff
@@ -2336,6 +2336,8 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-gci-new-gci-master-upgrade-cluster-new-parallel
 - name: ci-kubernetes-multicluster-ingress-test
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-multicluster-ingress-test
+- name: kubeflow-periodic-0-5-branch
```
Comment (richardsliu) [inline testgrid/config.yaml]: You may also need to update https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml#L5641

Comment (lluunn) [reply testgrid/config.yaml]: thanks! updated

**Comment by BenTheElder** [comment]
/unassign
/uncc
@jlewi you are the only approver (and no reviewers) in https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubeflow/OWNERS, please review or add more owners

**Comment by lluunn** [comment]
Thanks @BenTheElder , didn't notice we have the OWNERS file.

**Pull request merged**